packages/fastui/src/components/FormField.tsx

```diff
@@ -55,6 +55,7 @@ export const FormFieldInputComp: FC<FormFieldInputProps> = (props) => {
 
 interface FormFieldCheckboxProps extends BaseFormFieldProps {
   type: 'FormFieldCheckbox'
+  mode: 'checkbox'
```

Comment:
I think this should be
```suggestion
  mode: 'checkbox' | 'switch'
```